src/app/services/sms/sms_count.server.model.ts

```diff
@@ -125,6 +138,16 @@ const compileSmsCountModel = (db: Mongoose) => {
     return smsCount.save()
   }
 
+  SmsCountSchema.statics.retrieveFreeSmsCounts = async function (
+    userId: string,
+  ) {
+    return this.countDocuments({
+      'formAdmin.userId': userId,
+      smsType: SmsType.Verification,
+      isOnboardedAccount: false,
+    }).exec()
+  }
+
```

Comment:
Should we also add an `index` block here (even though it does nothing on Mongo Atlas) so people who fork this repository at least gets the index added.

```
SmsCountSchema.index({
      'formAdmin.userId': 1,
      smsType: 1,
      isOnboardedAccount: 1,
})
```

Reply:
mongoose [docs](https://mongoosejs.com/docs/guide.html#indexes) says that this incurs a perf hit and recommends disablign it using `autoIndex: false` in production. 

would this negatively impact our app as well or does atlas sidestep this?

Reply:
discussed offline; not too sure how atlas behaves w/ this so erring on safe side and not doing it.